apps/demo/stories/Button.stories.tsx

```diff
@@ -3,63 +3,67 @@ import React from "react";
 
 import {StorybookContainer} from "./StorybookContainer";
 
-export const ButtonDemo = () => {
+export const ButtonDemo = (props: Partial<ButtonProps>) => {
   return (
-    <Button color="primary" icon="plus" text="Button" onClick={() => console.info("clicked")} />
+    <Button iconName="plus" text="Button" onClick={() => console.info("clicked")} {...props} />
   );
 };
 
 export const AllColorButtons = (props: Partial<ButtonProps>) => {
   return (
     <StorybookContainer>
       <Box paddingY={1}>
-        <Button text="Default" onClick={() => console.info("clicked")} {...props} />
+        <Button text="Default/Primary" onClick={() => console.info("clicked")} {...props} />
       </Box>
       <Box paddingY={1}>
         <Button
-          color="primary"
-          text="Primary"
-          tooltip={{text: "Primary Button"}}
+          text="Secondary"
+          variant="secondary"
```

Comment:
This seems to be rendering a primary button still.

Reply:
sorry i'm not seeing that issue

Reply:
Ope sorry, it's the Modal secondary button that's rendering as primary

Reply:
okay. I do not see 2 primary buttons. Meaning they are rendering different on my end.
However the secondaryButton in Model is a button variant muted.
 Do you think i should be exclusive with the prop name
`secondaryButtonText` ->`mutedButtonText`
`secondaryButtonOnClick` -> `mutedButtonOnClick`
I'm not entirely sure if that's more of less clear as a prop